management-commands/create-muterole.md

```diff
@@ -3,6 +3,16 @@
 This command sets up a role to be used as the mute role for the server. It will edit the permissions of all channels to
 deny the role from sending messages.
 
+It will edit the following permissions form channels and set them to `false` for the role:
```

Comment:
```suggestion
This command sets up a role to be used as the mute role for the server. It will create permission overrides in every possible channels for the role, denying all the following permissions:
```

also, this needs to include a re-elaborations of the screenshots i sent on discord for the relevant conversation